demo/demo-tekton.sh

```diff
@@ -2,18 +2,23 @@
 #Assumes User is logged in to cluster
 set -euo pipefail
 
+app_name="basic-python-tekton"
+app_namespace="basic-python-tekton"
+
 Help()
 {
    # Display Help
    echo "Execute a tekton pipeline with various build types."
    echo
    echo "Syntax: scriptTemplate [-h|g|b|n]"
    echo "options:"
+   echo "a     application name, default ${app_name}"
+   echo "n     namespace to be used, default ${app_namespace}"
    echo "g     the git url"
    echo "r     git branch reference, use this for Pull Requests. e.g. refs/pull/587/head"
    echo "h     Print this Help."
    echo "b     build type [buildconfig, binary, s2i]"
-   echo "n     If set then no human interaction is required for subsequent deployments"
+   echo "i     If set then no human interaction is required for subsequent deployments"
```

Comment:
`i` traditionally means _explicitly interactive_, at least in things like `rm`. Can we use a long option?

Reply:
and you are totaly right, just wanted to use "n' for namespace and i is just something that was left unused ;)

Reply:
@KevinMGranger one more note. getopts in built-in shell accepts only short ones, there are "work-arounds" but I would rather not implement them for such simple script. Maybe easierst is to allow pass the argument to -i, e.g.:

```
-i [true/false]
```

where true is by default interactive, false is not?

Or different letter ?

Reply:
@KevinMGranger So I changed way it works. I really don't want to include long names as this is not supported by built-in getopts. We could use GNU getopt, but this would cause issues when working on different platforms (MacOS...).

So having said that, currently I have removed `-i` and decided to enter non-human mode whenever number of deployments `OR` time between deployments is passed to the script. This means that user wanted to have multiple deployments without pressing 1...